src/peergos/server/corenode/JdbcIpnsAndSocial.java

```diff
@@ -259,6 +260,19 @@ private synchronized void init(SqlSupplier commands) {
         }
     }
 
+    public CompletableFuture<Boolean> deletePointer(PublicKeyHash writingKey) {
+        try (Connection conn = getConnection();
+             PreparedStatement stmt = conn.prepareStatement(IPNS_REMOVE)) {
+            stmt.setString(1, new String(Base64.getEncoder().encode(writingKey.serialize())));
```

Comment:
This space seems to be associated with the usage of try-with-resources. I think it is positioned like the object initialization in the line above because this statement is part of the resources initialization, not the part of the try block.